MKOB/MKOB.pyw

```diff
@@ -35,6 +35,7 @@ import sys
 import kobwindow
 
 VERSION = "MorseKOB 4.0.6"
```

Comment:
I probably should have commented on this earlier (since this wasn't a current change), but I think 'VERSION' should just be '4.0.6'. The 'MorseKOB' should be added when printing/displaying it.

Reply:
Good suggestion. I'll change it the next time I touch the code.